scripts/rook.sh

```diff
@@ -204,6 +221,12 @@ create-block-pool)
 delete-block-pool)
 	delete_block_pool
 	;;
+create-block-ec-pool)
```

Comment:
case values (just strings) can contain `-`, so that is not a problem (as seen with other commands like `create-block-pool`).